frontend/src/pages/vault/stake/Stake.jsx

```diff
@@ -40,86 +40,89 @@ function Stake() {
+                        fill="none"
+                        xmlns="http://www.w3.org/2000/svg"
+                      >
+                        <path
+                          d="M6 9L12 15L18 9"
+                          stroke="currentColor"
+                          strokeWidth="2"
+                          strokeLinecap="round"
+                          strokeLinejoin="round"
+                        />
+                      </svg>
+                    </div>
 
-              <div className="network-dropdown">
-                {networks.map((network) => (
-                  <div key={network.name} className="network-option" onClick={() => handleChange(network)}>
-                    <img src={network.image} alt={network.name} className="network-icon" />
-                    <span>{network.name}</span>
+                    <div className="network-dropdown">
+                      {networks.map((network) => (
+                        <div key={network.name} className="network-option" onClick={() => handleChange(network)}>
+                          <img src={network.image} alt={network.name} className="network-icon" />
+                          <span>{network.name}</span>
+                        </div>
+                      ))}
+                    </div>
                   </div>
-                ))}
-              </div>
-            </div>
 
-            <div className="amount-input" aria-labelledby="amount-input-label">
-              <input
-                type="text"
-                id="amount-field"
-                value={amount}
-                onChange={handleAmountChange}
-                pattern="^\d*\.?\d*$"
-                className="amount-field"
-                aria-describedby="currency-symbol"
-                placeholder="0.00"
-              />
-              <span id="currency-symbol" className="currency">STRK</span>
-            </div>
+                  <div className="amount-input" aria-labelledby="amount-input-label">
+                    <input
+                      type="text"
+                      id="amount-field"
+                      value={amount}
+                      onChange={handleAmountChange}
+                      pattern="^\d*\.?\d*$"
+                      className="amount-field"
+                      aria-describedby="currency-symbol"
+                      placeholder="0.00"
+                    />
+                    <span id="currency-symbol" className="currency">STRK</span>
+                  </div>
 
-            <div className="apy-rate">$0.00 APY / year</div>
-            <GasFee />
-          </div>
+                  <div className="apy-rate">$0.00 APY / year</div>
+                  <GasFee />
+                </div>
 
-         </div>
-          <Button variant="secondary" size="lg" className="stake-button">
-            Stake
-          </Button>
+            </div>
+            <Button variant="secondary" size="lg" className="stake-button">
```

Comment:
button styles are broken now

Reply:
Do you mean all buttons or only this. The button style is making it difficult to style this